packages/base-styles/_mixins.scss

```diff
@@ -202,6 +202,11 @@
 	}
 }
 
+@mixin mark-style() {
+	font-weight: 700;
+	background: none;
+}
+
```

Comment:
--cc @jameskoster

Reply:
Is there a drawback to making it a mixin now? I don't know how likely it is that we use `mark` in other components, but it would be a shame to have the same styles spread across multiple stylesheets.

Reply:
I'm not really sure, but if we don't reuse it, we should probably have it inline for now.. We do want to sync with `LinkControl` styles as well, so this might not be a generic thing. We can always revisit 😄

Reply:
cc @ciampo

Reply:
From what I know, the `<mark>` HTML tag is used by:

- the [`TextHighlight` component](https://wordpress.github.io/gutenberg/?path=/story/components-texthighlight--default) (which uses the default browser styles)
- the [`Text` component](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-text--highlight) (which overrides the browser styles with a custom bg color / border radius / box shadow)


to highlight text matches.

(cc @mirka )